apps/io-wallet-web-func/src/infra/handlers/create-wallet-instance.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  WalletInstanceValid,
+  insertWalletInstance,
+  revokeUserWalletInstancesExceptOne,
+} from "@/wallet-instance";
+import * as H from "@pagopa/handler-kit";
+import { pipe } from "fp-ts/function";
+import * as RTE from "fp-ts/lib/ReaderTaskEither";
+
+export const CreateWalletInstanceHandler = H.of(
+  (walletInstance: WalletInstanceValid) =>
+    pipe(
+      insertWalletInstance(walletInstance),
+      RTE.chainW(() =>
+        revokeUserWalletInstancesExceptOne(
```

Comment:
I understand it was already introduced this way. But what do you think about renaming it to  `revokeUserWalletInstancesExceptLast` ? Maybe you could use the same function to get the latest wallet instance and eliminate the id dependency

Reply:
we could do it this way, but I would have to run a query to get an ID that I already have

Reply:
right!